generate_fixtures.py

```diff
@@ -25,4 +28,13 @@ def generate_fixtures():
     TUFTestFixtureThresholdTwoAttack.build()
 
 
+# Remove all previous fixtures.
+for f in glob.glob("fixtures/*/client"):
+    shutil.rmtree(f)
+for f in glob.glob("fixtures/*/server"):
+    shutil.rmtree(f)
+# Delete has files to ensure they are generated again.
+for f in glob.glob("fixtures/*/hash.txt"):
+    os.remove(f)
```

Comment:
IMHO it should be `FixtureBuilder`'s job to remove the existing hash file, maybe during `__init__`?

Reply:
I thought about that but in case where we are removing a fixture or where developer locally built the fixtures on a different branch that had a different set of fixtures we still want all the files under `fixtures/` to be deleted regardless of whether they are in the current set of fixtures.

Reply:
Is this actually needed, though? Because in `FixtureBuilder.__del__`, we're overwriting the existing hash file.

Reply:
See my comment about the hash.txt files themselves actually being hashed if they are present, even if empty.  We delete them all first because these would be the hashes for the fixtures before changes to builder scripts. Also there could be file for fixture we have deleted.